### PR TITLE
Added contents in FAQ.md.

### PR DESCRIPTION
**Pull request by Zubiii333**
**Notes for Reviewers**

This PR fixes #
Added three questions and answers.
Signed off the commits.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit fceb39a6b05f374eb901f0436d1ee59ec70dda21 at: https://652d4ef3d153d521b3151ceb--meshery-docs-preview.netlify.app

**Comment by abhijeetgauravm** [comment]
Hey @Zubiii333, please edit the PR description to include the issue it's fixing. This will make it easier for reviewers to review the PR.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/project/FAQ.md [hunk 1]
```diff
@@ -36,6 +36,20 @@ _Popular Installers:_
...
+
+**Answer:** _No, as the cloud native management plane, Meshery provisions and manages the lifecycle of service meshes and all other kinds of cloud native infrastructure._
+
+#### Is Meshery open source project?
+
+**Answer:** _Yes, Meshery is a Cloud Native Computing Foundation (CNCF) project and is licensed under Apache v2. As the cloud native management plane, Meshery is an extensible platform, offering multiple extension points within which users and partners can customize and extend Meshery's functionality._
+
+#### Question: Why should I use Meshery?
+
+**Answer:** _Meshery is an open source, vendor neutral project that facilitates testing across meshes. Other performance benchmark tools are not packaged into a mesh testing utility, but are only load-generators unto their own._
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply docs/pages/project/FAQ.md:49]: @Zubiii333 message when you have changed this.

**Comment by l5io** [comment]
🚀 Preview for commit 42900a415c530e6e089ce84c3ced0d4780fa79be at: https://652ebc007008be35274e8f92--meshery-docs-preview.netlify.app

**Comment by saurabh100ni** [comment]
Thanks for raising the PR, let's Discuss this in the Meshery Development Call today,
Please add this as an agenda in the Doc : [Doc](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit)

File: docs/pages/project/FAQ.md [hunk 2]
```diff
@@ -36,6 +36,20 @@ _Popular Installers:_
...
+
+**Answer:** _No, as the cloud native management plane, Meshery provisions and manages the lifecycle of service meshes and all other kinds of cloud native infrastructure._
+
+#### Is Meshery open source project?
+
+**Answer:** _Yes, Meshery is a Cloud Native Computing Foundation (CNCF) project and is licensed under Apache v2. As the cloud native management plane, Meshery is an extensible platform, offering multiple extension points within which users and partners can customize and extend Meshery's functionality._
+
+#### Question: Why should I use Meshery?
+
+**Answer:** _Meshery is a powerful tool for managing ​service meshes and ​Kubernetes infrastructure. It seamlessly integrates with different service meshes and offers extensibility through dedicated adapters for each supported service mesh. With Meshery, you can easily configure your environment and service meshes, upload and manage applications, and analyze performance._
```
Comment (leecalcote) [inline docs/pages/project/FAQ.md:49]:
```suggestion
**Answer:** _Meshery is a powerful tool for managing ​Kubernetes infrastructure. It seamlessly integrates with different hundreds of tools and offers extensibility through many different [extension points]({{site.baseurl}}/extensibility/#extension-points). With Meshery, you can easily discover your environment, collaboratively manage multiple Kubernetes clusters, connect your Git and Helm repos, and analyze app and infra performance._
```

File: docs/pages/project/FAQ.md [hunk 3]
```diff
@@ -36,6 +36,20 @@ _Popular Installers:_
 - _Kubernetes user._
 - _[Scoop user](https://github.com/meshery/scoop-bucket)._ 
 
+#### Question: Is Meshery a service mesh?
+
+**Answer:** _No, as the cloud native management plane, Meshery provisions and manages the lifecycle of service meshes and all other kinds of cloud native infrastructure._
```
Comment (leecalcote) [inline docs/pages/project/FAQ.md:41]:
```suggestion
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**